apps/src/lab2/lab2Redux.ts

```diff
@@ -284,7 +287,9 @@ export const setUpWithLevel = createAsyncThunk<
 
 // If any load is currently in progress.
 export const isLabLoading = (state: {lab: LabState}) =>
-  state.lab.isLoadingProjectOrLevel || state.lab.isLoading;
+  state.lab.isLoadingProjectOrLevel ||
+  state.lab.isLoading ||
+  state.lab.isLoadingTheme;
```

Comment:
Assuming these are mutually exclusive, should we group them into a single variable?

Reply:
As in have 1 redux value? I think that won't work because we want to wait until all loads are done. So if the theme finishes loading but we haven't yet set `isLoading` to false, we still want to wait for `isLoading` to be set to false before declaring the lab as loaded.